misk/src/main/kotlin/misk/web/metadata/ConfigMetadataAction.kt

```diff
@@ -55,7 +56,7 @@ class ConfigMetadataAction @Inject constructor(
       environment: Environment,
       config: Config
     ): Map<String, String?> {
-      val rawYamlFiles = MiskConfig.loadConfigYamlMap(appName, environment.name, listOf())
+      val rawYamlFiles = MiskConfig.loadConfigYamlMap(appName, Env(environment.name), listOf())
```

Comment:
This type of awkwardness will go away since all the functions taking `Environment` will be replaced soon with `Env`-only functions.